centreon/lang/de_DE.UTF-8/LC_MESSAGES/messages.po

```diff
@@ -15751,6 +15751,39 @@ msgstr ""
 # msgid 'Include services for these hosts'
 # msgstr ""
 
+# msgid "Select a widget type"
+# msgstr ""
+
+# msgid "No widget found"
+# msgstr ""
+
+# msgid "Delete a widget"
+# msgstr ""
+
+# msgid "Do you want to delete this widget?"
+# msgstr ""
+
+# msgid "Widget library"
```

Comment:
```suggestion
# msgid "Widgets library"
```

Reply:
Is it "widgets" or "widget" because @cg-tw told me to remove the "s" from "widgets"

Reply:
it is widget

Reply:
ok!